### PR TITLE
Run functional librosa compatibility test on CUDA as well

### PR DESCRIPTION
**Pull request by mthrok**
This PR parameterizes the device placement of the librosa compatibility test for functional, so that the tests run on GPU as well.

### CODE REVIEW

File: test/torchaudio_unittest/functional/librosa_compatibility_cuda_test.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from .librosa_compatibility_test_impl import Functional, FunctionalComplex
+
+
+@skipIfNoCuda
+class TestFunctionalCUDA(Functional, PytorchTestCase):
+    device = 'cuda'
+
+
+@skipIfNoCuda
+class TestFunctionalComplexCPU(FunctionalComplex, PytorchTestCase):
```
Comment (carolineechen) [inline test/torchaudio_unittest/functional/librosa_compatibility_cuda_test.py:11]: do you mean for this to be `TestFunctionalComplexCUDA` and `device = 'cuda'`?

File: test/torchaudio_unittest/functional/librosa_compatibility_test_impl.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        win_length = n_fft
+        hop_length = n_fft // 4
+        window = torch.hann_window(win_length)
+        power = 1
+        # GriffinLim params
+        n_iter = 8
+
+        waveform = get_whitenoise(device=self.device, dtype=self.dtype)
+        specgram = get_spectrogram(
+            waveform, n_fft=n_fft, hop_length=hop_length, power=1,
```
Comment (carolineechen) [inline test/torchaudio_unittest/functional/librosa_compatibility_test_impl.py:43]: nit: power=power

File: test/torchaudio_unittest/functional/librosa_compatibility_cuda_test.py (hunk 1 above)
Comment (mthrok) [reply test/torchaudio_unittest/functional/librosa_compatibility_cuda_test.py:11]: Yes, I forgot to fix it after copy-pasting the code from CPU version. Thanks for catching.

**Review by carolineechen — APPROVED** [review]
Looks good!

**Pull request merged**